library/core/src/num/f32.rs

```diff
@@ -357,38 +357,76 @@ impl f32 {
     pub const MANTISSA_DIGITS: u32 = 24;
 
     /// Approximate number of significant digits in base 10.
+    ///
+    /// This is the maximum <i>x</i> such that any decimal number with <i>x</i>
```

Comment:
I'm used to write `<i>x</i>` instead of `*x*` because `*x*` is converted to `<em>x</em>`, and while that *does* make semantic sense to emphasize words such as "does" in this sentence, it does not make semantic sense in mathematical italics. Having said that, I don't feel strongly about this so if you think I should make the changes anyway I will.

Reply:
Fair point. Proceed however you would like.